spec/features/choices_feature_spec.rb

```diff
@@ -1,20 +1,21 @@
 require 'rails_helper'
 
 describe 'Choices Features' do
+  let(:event){ create :event, user: create(:user) }
+
   before :each do
-    user = create :user
-    sign_in_as(user)
-    @event = create(:event, user: user)
-    @question = create(:question, content: "Is the Sky Blue?", event: @event)
+    sign_in_as(event.user)
+    @question = create(:question, content: "Is the Sky Blue?", event: event)
```

Comment:
WHY ARE YOU ALWAYS PUTTING CONTENT IN THE FACTORIES IT IS ALREADY THERE !!!!!!!!!!!!